src/mumble/AudioInput.cpp

```diff
@@ -1072,8 +1075,12 @@ void AudioInput::encodeAudioFrame(AudioChunk chunk) {
 	bIsSpeech = bIsSpeech || (Global::get().iPushToTalk > 0);
 
 	ClientUser *p = ClientUser::get(Global::get().uiSession);
+	bool bTalkingWhenMuted = false;
 	if (Global::get().s.bMute || ((Global::get().s.lmLoopMode != Settings::Local) && p && (p->bMute || p->bSuppress)) || Global::get().bPushToMute
 		|| (voiceTargetID < 0)) {
+		if (bIsSpeech) {
+			bTalkingWhenMuted = true;
+		}
```

Comment:
This could be shortened as
```suggestion
		bTalkingWhenMuted = bIsSpeech;
```
The `if` makes the semantics a bit clearer though, so in the end I think I don't mind which version is used (I assume the extra branch will be optimized away anyways). Your call

Reply:
I just found what is arguably a bug: If we are deafened, the mute cue should not play. So I added that to the expression, PTAL.